packages/atomic/src/components/common/facets/facet-header/facet-header.tsx

```diff
@@ -34,7 +34,7 @@ export const FacetHeader: FunctionalComponent<FacetHeaderProps> = (props) => {
       style="text-transparent"
       part="label-button"
       class="flex font-bold justify-between w-full py-1 px-2 text-lg rounded-none"
-      title={props.isCollapsed ? expandFacet : collapseFacet}
+      ariaLabel={props.isCollapsed ? expandFacet : collapseFacet}
```

Comment:
⚠️ Small but interesting change.

there's an order of things for an 'accessible name' (as defined by ARIA).
https://russmaxdesign.github.io/accessible-forms/accessible-name-button01.html

So, the title was not used by screen readers notably.

I elected to replace the ariaLabel by title so that screenreaders -and my tests- pick up the string

Reply:
The only issue with adding both is that screen reader that support both may read the aria-label value and then the title value. For example I tried with voiceover utility and it does read both.

Reply:
> The only issue with adding both is that screen reader that support both may read the aria-label value and then the title value. For example I tried with voiceover utility and it does read both.

Should not tho according to W3C:
1. https://www.w3.org/TR/html-aam-1.0/#input-type-text-input-type-password-input-type-number-input-type-search-input-type-tel-input-type-email-input-type-url-and-textarea-element-accessible-name-computation
2. https://www.w3.org/TR/accname-1.2/#computation-steps

It should follow 2Dii resolution path.